docs/how_to_contribute.md

```diff
@@ -163,13 +192,13 @@ Thus `https://www.latimes.com/news-sitemap.xml` is a Google News Index Map.
 
 #### Finishing the Publisher Specification
 
-To complete the publisher specification, specify at least one of the following parameters:
+1. Sometimes spanning the entire site means spanning the entire site, so sitemaps can include a lot of noise like sitemaps pointing to a collection of tags or authors, etc.
```

Comment:
This sentence contains a duplicate definition

Reply:
I've changed this.